docroot/themes/humsci/humsci_basic/src/scss/tools/_mixins.general.scss

```diff
@@ -96,3 +96,12 @@
     }
   }
 }
+
+@mixin hb-divider {
+  width: 100%;
+
+  @include hb-colorful {
+    border-bottom: $hb-thin-border;
+    @include hb-global-color('border-color', 'gray');
+  }
+}
```

Comment:
Let's update this divider class to apply to views, like we discussed in our call. Also updating the structured card as Alex described. 👍